globals/matchmaking_globals.go

```diff
@@ -15,6 +16,7 @@ type MatchmakingManager struct {
 	Mutex                    *sync.RWMutex
 	GetUserFriendPIDs        func(pid uint32) []uint32
 	GetDetailedGatheringByID func(manager *MatchmakingManager, gatheringID uint32) (types.RVType, string, *nex.Error)
+	CanJoinMatchmakeSession  func(manager *MatchmakingManager, pid *types.PID, matchmakeSession *match_making_types.MatchmakeSession) *nex.Error
```

Comment:
This override should be added on the MatchmakeExtension common protocol, not here. The `MatchmakingManager` should not include any NEX-specific overrides (`match-making` and `match-making-ext` come from QRV, whereas `matchmake-extension` and the `MatchmakeSession` are specific to NEX).

The override should be done in the places where `CanJoinMatchmakeSession` is called, like for the other overrides present in the MatchmakeExtension common protocol